## Description: Let adb detect devices from BQ.
## Origin/Author: Jani Monoses <jani.monoses@canonical.com>
---
# system/core/adb/usb_vendors.c |    3 +++
# 1 file changed, 3 insertions(+)
#
--- a/system/core/adb/usb_vendors.c
+++ b/system/core/adb/usb_vendors.c
@@ -51,6 +51,8 @@
 #define VENDOR_ID_ARCHOS        0x0E79
 // Asus's USB Vendor ID
 #define VENDOR_ID_ASUS          0x0b05
+// BQ's USB Vendor ID
+#define VENDOR_ID_BQ            0x2a47
 // BYD's USB Vendor ID
 #define VENDOR_ID_BYD           0x1D91
 // Compal's USB Vendor ID
@@ -206,6 +208,7 @@ int builtInVendorIds[] = {
     VENDOR_ID_ANYDATA,
     VENDOR_ID_ARCHOS,
     VENDOR_ID_ASUS,
+    VENDOR_ID_BQ,
     VENDOR_ID_BYD,
     VENDOR_ID_COMPAL,
     VENDOR_ID_COMPALCOMM,
